celestia/chain.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "celestia",
+  "chain_id": "celestia",
+  "pretty_name": "Celestia",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "celestia",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
https://github.com/satoshilabs/slips/pull/1625 please let me know if there's anything else, thanks!